app/jobs/risc_delivery_job.rb

```diff
@@ -102,18 +102,13 @@ def faraday
     end
   end
 
-  def inline?
```

Comment:
if we're following through with this we should also update the logic in `http_push.rb` and remove the `risc_notifications_active_job_enabled` feature flag as well

Reply:
Right, I saw that when the tests failed on CI. Looking into it.